tools/Get-NuGetTool.ps1

```diff
@@ -0,0 +1,15 @@
+<#
```

Comment:
For now powershell is good, but it's not my cup of tea. In the future we might use cakebuild instead because it's easier for devs like me to write C# scripts.